designs/osc_high_availability/OSC_High_Availability.md

```diff
@@ -0,0 +1,98 @@
+# OSC High Availability
```

Comment:
change the file name to use lower case only for consistency with the other files